### ISSUE TITLE
added container stats method

### ISSUE THREAD

**Issue by TomHutter** [swipely/docker-api#408]
I extended the Docker::Container class with method:

```
# Return a Hash with the container resource statistics
# see: https://docs.docker.com/engine/reference/api/docker_remote_api_v1.23/#get-container-stats-based-on-resource-usage/vol/
def stats(opts = {})
    opts[:stream] = false # do not stream, pull only once  
    Docker::Util.parse_json(connection.get(path_for(:stats), opts))
end
```

You may want to add this to the trunk.

**Comment by MichaelSp** [comment]
related to #250 and #384 ?

**Comment by randywallace** [comment]
any progress on this?  I had the feature working a while back, and this is so easy to implement and doesn't break existing functionality... I've been using it off a fork for quite a while but would really like to see it make it into mainstream

**Comment by tlunter** [comment]
Released in v1.29.0
